app/Repositories/Summit/DoctrineSummitEventRepository.php

```diff
@@ -616,12 +697,61 @@ protected function getOrderMappings()
 END
 SQL,
             'selection_plan' => 'selp.name',
-            'actions' => 'HIDDEN_COMPLETED_ACTIONS_COUNT'
+            'actions' => 'HIDDEN_COMPLETED_ACTIONS_COUNT',
             /*
             'event_type_capacity' => <<<SQL
 SQL,
             'speakers' => <<<SQL
 SQL,*/
+            'review_status' => <<<SQL
+    CASE
+    WHEN p is null THEN ''
+	WHEN selp IS NULL OR (p.status <> 'Received' AND p.status <> 'Accepted') THEN 'NoSubmitted'
+	WHEN (p.status = 'Received' OR p.status = 'Accepted') AND
+		selp IS NOT null AND (
+		(selp.submission_lock_down_presentation_status_date IS NOT NULL AND selp.submission_lock_down_presentation_status_date > '$now_utc') OR 
+		(
+			(selp.submission_begin_date > '$now_utc' OR selp.submission_end_date < '$now_utc') 
+			AND 
+			(selp.selection_begin_date <= '$now_utc' AND selp.selection_end_date >= '$now_utc')
+		)
+	) THEN 'InReview'
+	WHEN (p.status = 'Received' OR p.status = 'Accepted') AND e.published = 1 AND
+		selp IS NOT null AND ( 
+		(selp.submission_lock_down_presentation_status_date IS NULL OR selp.submission_lock_down_presentation_status_date <= '$now_utc') AND 
+		(
+			(selp.submission_begin_date <= '$now_utc' AND selp.submission_end_date >= '$now_utc') 
+			OR 
+			(selp.selection_begin_date > '$now_utc' OR selp.selection_end_date < '$now_utc')
+		)
+	) THEN 'Published'
+    WHEN (p.status = 'Received' OR p.status = 'Accepted') AND e.published = 0 AND 
+        (selp.selection_begin_date > '$now_utc' OR selp.selection_end_date < '$now_utc') 
```

Comment:
u need to check here the proper conditions for
NOT lockDownPeriod AND ( submissionOpen OR selectionClosed )